image/verify/src/verifier.rs

```diff
@@ -589,6 +597,18 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
             .sha384_digest(range.start, range.len() as u32)
             .map_err(|_| err_u32!(VendorPubKeyDigestFailure))
     }
+
+    /// Calculates the effective fuse SVN.
+    ///
+    /// If anti-rollback is disabled, the effective fuse-SVN is zero.
+    /// Otherwise, it is SVN-fuses.
+    fn effective_fuse_svn(fuse_svn: u32, anti_rollback_disable: bool) -> u32 {
+        if anti_rollback_disable {
+            0_u32
+        } else {
```

Comment:
I`m not sure I understand how this works. Per spec Runtime SVN is 128 bits, and FMC is 32 bits.
Also it gets later downcast to a u8

Reply:
The bits start out in a unary representation, but the driver [converts](https://github.com/chipsalliance/caliptra-sw/blob/2e26d715dcbfa92c1e5e7eb3bcd2cda760db9792/drivers/src/fuse_bank.rs#L206-L236) them to numerical.

I picked u32 as the type just to mirror what's already present at this layer, but we could just change this to u8 here, to cut down on the number of different representations.

Reply:
I think sticking with either u32 or u8 might prevent any room for confusion...but I`m ok with that. Thanks

Reply:
Ack, will refactor to use u8 consistently in a follow-up PR (don't have bandwidth this week and I understand there's work that's blocked on this).